tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1546,6 +1579,18 @@ std::ostream& operator<<(std::ostream& os, const TestParamBase& p) {
   return os;
 }
 
+// Printing vector with the numbers of type T which defines tensor or shape
+template<typename T>
+void out_vector(const std::vector<T> &vector, absl::string_view pComment,
+                absl::string_view name, absl::string_view type="") {
```

Comment:
I understand we have made changes here, can we refine here one more time like this:

Change this routine to return a string for the output,  and name it get_debug_string.

Fix the callsites to use it like this:
if (VLOG_IS_ON(2) {
   VLOG(2) << get_debug_string_for_vector(...)
}

Reply:
Done